Sources/MapboxMaps/Style/Style.swift

```diff
@@ -5,9 +5,19 @@ import MapboxMapsFoundation
 
 //swiftlint:disable file_length
 public class Style {
+
+    public static let defaultURI = StyleURI.streets
```

Comment:
does this need to be public?

Reply:
I think it does since it's used as a default value.